tasks/tasks_test.go

```diff
@@ -119,7 +119,11 @@ func (t *SerialTests) TestCreateTask() {
 
 	// queue up a transfer task between two phony databases
 	taskId, err := Create(Specification{
-		UserInfo: auth.UserInfo{
+		Client: auth.Client{
+			Name:  "Joe-bob",
+			Orcid: "1234-5678-9012-3456",
+		},
+		User: auth.User{
```

Comment:
I don't know if at some future point, you want to write tests that ensure that the client orcid can auth transfers if the user orcid can't, etc.

Reply:
We haven't worked out all the ORCID machinery yet, because the effort on all the other moving parts is so diffuse at the moment. When I have a better idea of what we want to do, I'll make sure all the desired code paths are tested.